appserver/appclient/client/acc/src/main/java/org/glassfish/appclient/client/jws/boot/JWSACCMain.java

```diff
@@ -40,27 +40,22 @@
 
 package org.glassfish.appclient.client.jws.boot;
 
-import java.io.BufferedWriter;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.OutputStreamWriter;
+import org.glassfish.appclient.client.acc.AppClientContainer;
+import org.glassfish.appclient.client.acc.JWSACCClassLoader;
+import org.glassfish.appclient.common.Util;
+
+import javax.swing.*;
```

Comment:
Minor point, but is this necessary? Aren't we only using one class from here?

Reply:
done